src/ast.zig

```diff
@@ -572,7 +572,9 @@ pub fn lastToken(tree: Ast, node: Ast.Node.Index) Ast.TokenIndex {
         => {
             end_offset += 2; // for the comma/semicolon + rparen/rbrace
             const params = tree.extraData(datas[n].rhs, Node.SubRange);
-            std.debug.assert(params.end > params.start);
+            if (params.end - params.start == 0) {
```

Comment:
Why not `params.end == params.start`?

Reply:
I just copied what was used in the switch case for `.call` and `.tagged_union_enum_tag` above it. Its the same in the zig codebase. My only guess is because it adds a safety check for integer overflow.